eng/liveBuilds.targets

```diff
@@ -1,10 +1,10 @@
 <Project>
 
   <PropertyGroup>
-    <CoreCLROSGroup Condition="'$(CoreCLROSGroup)' == ''">$(OSGroup)</CoreCLROSGroup>
-    <CoreCLRConfiguration Condition="'$(CoreCLRConfiguration)' == ''">$(Configuration)</CoreCLRConfiguration>
-    <LibrariesOSGroup Condition="'$(LibrariesOSGroup)' == ''">$(OSGroup)</LibrariesOSGroup>
-    <LibrariesConfiguration Condition="'$(LibrariesConfiguration)' == ''">$(Configuration)</LibrariesConfiguration>
+    <CoreCLROSGroup Condition="'$(CoreCLROSGroup)' == ''">$(BuildOS)</CoreCLROSGroup>
```

Comment:
Will build OS be available when building `CoreCLR`?

Reply:
yes BuildOS was already being used by coreclr to build and place the assets.

Reply:
I think it was using `OSGroup`:

https://github.com/dotnet/runtime/pull/1787/files/ecc1f155796aecf4651ca9811fee9ed4f2ab9d3c#diff-03d5343f21842e18dadb225fc70c2cc5L4

I believe these targets are not going through the new config system, right?

Reply:
The way I understand its working is the coreclr is built using https://github.com/dotnet/runtime/blob/master/src/coreclr/dir.common.props#L24
And we are coreclrOSGroup to get the assets from that path